### PR TITLE
relayer2 deployment

### PR DESCRIPTION
**Pull request by akeyder**
changes w.r.t deployment of new Aurora Relayer: https://github.com/aurora-is-near/relayer2-public
- docker images of the relayer2 and refiner are now tagged as
 - `nearaurora/srpc2-refiner`
 - `nearaurora/srpc2-relayer`
- after installation all artifacts (config, data, etc) can be found at `<repo dir>/srpc2/` see README for more details
- `contrib/build/build.sh` can be used to build and push images to docker repository. It expects two options `-v {relayer version tag}` `-r {refiner version tag}`

### CODE REVIEW

File: contrib/config/relayer/mainnet.yaml [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+  backupDir: /data/backup/v2_mainnet_aurora_blocks
+  backupNamePrefix: v2_mainnet_aurora_blocks_
+  from: 0
+prehistoryIndexer:
+  indexFromPrehistory: false
+  prehistoryHeight: 37157758
+  from: 0
+  to: 37157757               # cannot be higher than prehistoryHeight
+  batchSize: 10000
+  chainId: 1313161554
```
Comment (tolga-coplu) [inline contrib/config/relayer/mainnet.yaml:83]: `chainId` field was renamed as `prehistoryChainId`. 
Can you please update accordingly

File: contrib/config/relayer/testnet.yaml [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+  backupDir: /data/backup/v2_testnet_aurora_blocks
+  backupNamePrefix: v2_testnet_aurora_blocks_
+  from: 0
+prehistoryIndexer:
+  indexFromPrehistory: false
+  prehistoryHeight: 47354108
+  from: 0
+  to: 47354107                # cannot be higher than prehistoryHeight
+  batchSize: 10000
+  chainId: 1313161555
```
Comment (tolga-coplu) [inline contrib/config/relayer/testnet.yaml:83]: `chainId` field was renamed as `prehistoryChainId`. 
Can you please update accordingly

File: contrib/config/relayer/mainnet.yaml (hunk 1 above)
Comment (akeyder) [reply contrib/config/relayer/mainnet.yaml]: updated, thanks!

File: contrib/config/relayer/testnet.yaml (hunk 2 above)
Comment (akeyder) [reply contrib/config/relayer/testnet.yaml]: updated, thanks!

**Review by tolga-coplu — APPROVED** [review]
LGTM

**Pull request merged**